manifests/base/gateway-controller/gateway-controller-configmap.yaml

```diff
@@ -1,13 +1,24 @@
-# When installed as cluster-wide, the controller watches all namespaces by default.
-# To watch gateways created in a specific namespace only, add following key:
-# data:
-#  config: |
-#    instanceID: argo-events
-#    namespace: <your_namespace>
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: gateway-controller-configmap
 data:
-  config: |
```

Comment:
This needs to be removed in the release, otherwise it causes issues when users apply upgraded manifests.